ui/hooks/gasFeeInput/useTransactionFunctions.js

```diff
@@ -164,17 +168,30 @@ export const useTransactionFunctions = ({
         maxPriorityFeePerGas,
       } = transaction.previousGas || transaction.txParams;
 
+      const newMaxPriorityFeePerGas =
+        maxPriorityFeePerGas === '0x0'
```

Comment:
I see a case in our codebase where the value is `0x00` or `0x0` (https://github.com/MetaMask/metamask-extension/blob/develop/ui/selectors/selectors.js#L1381) ; do we need to check for both?

Reply:
Am I misremembering, or did we have a constant somewhere for 0 hex values?

Reply:
BigNumber has a .isZero method (@DDDDDanica suggested we add this to Numeric.ts) we could add that and then instantiate the number and use the .isZero check instead of relying on exact string comparison.

Reply:
```
new Numeric(maxPriorityFeePerGas, 16).isZero ?
```

Reply:
yep, good suggestion

Reply:
ah, `Numeric` isn't on develop yet

Reply:
oh, yes it is

Reply:
ah, but `isZero` is not part of `Numeric` yet

Reply:
used bignumber instead, in 783d31067